server/src/main/java/com/ca/lsp/cobol/service/TextDocumentSyncType.java

```diff
@@ -20,5 +20,6 @@
  */
 public enum TextDocumentSyncType {
```

Comment:
With the new value defined here, I think is time to find a more appropriate value for that enum. In past was named ```TextDocumentSyncType``` because was the reference in the LSP protocol specification for ```DID_OPEN``` and ```DID_CHANGE```

Reply:
It may be replaced with according methods in `CopybookProcessingMode`

Reply:
I've deleted this enum.